config/routes.rb

```diff
@@ -10,7 +10,16 @@
   get '/login', to: 'sessions#new'
   post '/login', to: 'sessions#create'
   delete '/logout', to: 'sessions#destroy'
+
   post 'cart_items/increase', to: 'cart_items#increase'
   post 'cart_items/decrease', to: 'cart_items#decrease'
 
+
+  namespace :admin do
+    get '/', to: 'dashboard#index'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.